collections/Laud_Misc/MS_Laud_Misc_306.xml

```diff
@@ -83,35 +133,299 @@
+                              <incipit>Hodierni misterii sacramentum fratres karissimi sicut credidistis semper et creditis</incipit>
+                           </msItem>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 156r–157v)</locus>
+                        <title type="desc">Three sermons for the feast of St Stephen (26 December)</title>
+                        <msItem>
+                           <msItem>
+                              <locus>(fol. 156r)</locus>
+                              <rubric>Sermo de sancto Stephano</rubric>
+                              <incipit>Hesterno die natalem habuimus domini salvatoris</incipit>
+                           </msItem>
+                           <msItem>
+                              <locus>(fols. 156r–157r)</locus>
+                              <rubric>Item de eodem</rubric>
+                              <incipit>Donet mihi dominus dicere salubriter qui donavit Stephano tanta dicere</incipit>
+                           </msItem>
+                           <msItem>
+                              <locus>(fol. 157r–v)</locus>
+                              <rubric>Sermo unde supra</rubric>
+                              <incipit>Natalem domini hesterna die celebravimus natalem servi hodie celebramus</incipit>
+                           </msItem>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fols. 157v–158v)</locus>
+                        <msItem>
+                           <title type="desc">Two sermons for the feast of the Holy Innocents (28 December)</title>
+                           <msItem>
+                              <locus>(fols. 157v–158r)</locus>
+                              <rubric>Sermo de innocentibus</rubric>
+                              <incipit>Christo igitur secundum prophetię fidem nato in Bethleem Iudę temporibus Herodis</incipit>
+                           </msItem>
+                           <msItem>
+                              <locus>(fol. 158r–v)</locus>
+                              <rubric>Item sermo ut supra</rubric>
+                              <incipit>Quocienscumque fratres karissimi sanctorum martyria celebramus</incipit>
+                           </msItem>
+                        </msItem>
+                     </msItem>
+                     <msItem>
+                        <locus>(fol. 159r–v)</locus>
+                        <title type="desc">Sermon for Epiphany (6 January)</title>
+                        <rubric>Sermo Leonis pape de epiphania domini</rubric>
+                        <incipit>Dies dilectissimi quo primum gentibus salvator mundi apparuit</incipit>
+                        <explicit defective="true">serit discordias. ascendit iras. incitat lin</explicit>
+                     </msItem>
+                  </msContents>
```

Comment:
some of the authors could be marked up, but perhaps let's leave this for a fuller revision of the textual content if time allows later